### ISSUE TITLE
1.0.5.6 bugs

### ISSUE THREAD

**Issue by aviator64** [Stephan-S/FS19_AutoDrive#474]
A few things I've noticed that only started happening in the last few releases..

1) Park on field does not work. Tractor always goes to park after unloading a combine, even when nowhere near the fruit.

2) When loading from a silo and "drive on when silo empty" is set, vehicle does not drive on but waits at the empty silo. Clicking Continue does not work either.

3) When using Load mode to fill a spreader/sprayer, as the vehicle approaches the field destination marker, the cover opens again. When control is passed back to Courseplay the vehicle then goes back to work with its cover open.

4) A tractor towing a sprayer, with an extra tank on the front for the same liquid - the front tank does not get filled in Load mode.

**Comment by Stephan-S** [comment]
Hey thanks for the feedback.
To 1.) I think this should be fixed. The problem was that the vehicle specific setting 'Park in field' was not used vehicle specific and was always used from the last vehicle that had the menu open. So this should work as expected again

2) Yeah I noticed that too. Will have to look into that. Thanks

3) Fixed a few hours ago. (Hopefully, haven't had time to test it)

4) Yeah, I disregard fillUnit with less than 1000 liters. This has to be tweaked. It was required when I added all the different fill units of the actual vehicle and not just the trailer itself, to support self-driving sprayers and trucks etc.

Best regards,
Stephan
